dialogflow_v2beta1/gapic/enums.py

```diff
@@ -322,6 +406,148 @@ class Platform(enum.IntEnum):
             VIBER = 7
             ACTIONS_ON_GOOGLE = 8
             TELEPHONY = 10
+            GOOGLE_HANGOUTS = 11
+
+
+        class RbmCarouselCard(object):
```

Comment:
Add RbmCarouselCard, RbmStandaloneCard, RbmCardContent, MediaContent, BrowseCarouselCard, ColumnProperties enums to v2beta1